Documentation/gettingstarted/hubble-relay-enable.rst

```diff
@@ -0,0 +1,88 @@
+Enable Hubble Relay for Cluster-Wide Visibility
+===============================================
+
+Hubble is the component for observability in Cilium. By default, Hubble
+provides visibility for traffic observed by the local agent only. For
+cluster-wide visibility, Hubble Relay needs to be enabled. For more information
+about observability in Cilium, see the :ref:`concepts_observability` section.
+
+The Hubble API endpoint of each agent needs to be configured to listen on TCP
+to allow Hubble Relay to communicate with all instances in the cluster. This endpoint is secured 
+using mutual TLS (mTLS) by default to only allow access to Hubble Relay.
+
+To enable Hubble Relay, use the following Helm upgrade command on your existing
+installation (Cilium agent pods will be restarted in the process):
+
+.. parsed-literal::
+
+   helm upgrade cilium |CHART_RELEASE| \\
+      --namespace $CILIUM_NAMESPACE \\
+      --reuse-values \\
+      --set hubble.listenAddress=":4244" \\
+      --set hubble.relay.enabled=true
+
+If the Cilium agent pods were not restarted, it should be triggered manually as
+it is necessary for the Cilium agent to pick up the ConfigMap changes.
+
+.. code:: bash
+
+   kubectl rollout restart -n $CILIUM_NAMESPACE ds/cilium
```

Comment:
Not sure I understand this. Are there cases where `helm upgrade` may not restart the Cilium pods? Isn't the validation step below enough to catch corner cases?

Reply:
These instructions are not new. However, when testing, I noticed that the `helm upgrade` command automatically restarts the Cilium pods but thought that the rollout restart instructions were there for a reason. However, as you point out, it might not be necessary thus I have now removed it.